update dex docs (#1362)
giantswarm/docs: src/content/advanced/configure-dex-in-your-cluster/index.md

We'll use the [app platform](https://docs.giantswarm.io/app-platform/) to deploy the app, as it allows us to deploy apps across workload clusters using a single API endpoint. In this example, we create an `App` custom resource (CR) with the parameters to install our [`dex-app`](https://github.com/giantswarm/dex-app) in the desired cluster, and a `ConfigMap` with the configuration values.

The `connectorConfig` format can look different depending on the oidc provider you want to use. Some examples can be found below. 

> Detailed upstream documentation on all currently available connectors and their respective configuration is available [here](https://dexidp.io/docs/connectors/).

Reviewer: "here" is not an ideal link text. Better use something that will give the reader a clue where they will end up.

```suggestion
Details on all connectors and their respective configuration is available in the [Dex documentation](https://dexidp.io/docs/connectors/).
```

> You can use more than one connector, but they need to have a different `id` value. We advice to use `- id: customer` for your primary connector.

Reviewer 2: Sorry a grammatical correction here : We advise you to use...

  namespace: <CLUSTER>
spec:
  catalog: giantswarm
  kubeConfig:
    context:
      name: <CLUSTER>
    secret:
      name: <CLUSTER>-kubeconfig
      namespace: <CLUSTER>
  name: dex-app
  namespace: dex
  userConfig:
    configMap:
      name: dex-app-user-values
      namespace: <CLUSTER>
  version: 1.22.2
```

> __Note__: When applying the example in the snippet above, please replace the `<CLUSTER>` placeholder with the name of the workload cluster you are configuring.

Reviewer 2: ... the workload cluster which you are configuring.
